infoschema/tables.go

```diff
@@ -1997,6 +2011,81 @@ func dataForTiDBClusterInfo(ctx sessionctx.Context) ([][]types.Datum, error) {
 	return rows, nil
 }
 
+type addressWithType struct {
+	addr string
+	// TiDB/PD/TiKV
+	tp string
+}
+
+func dataForClusterLoadInfo() ([][]types.Datum, error) {
+	serversInfo, err := infosync.GetAllServerInfo(context.Background())
+	if err != nil {
+		return nil, err
+	}
+
+	addrWithTps := make([]addressWithType, 0, len(serversInfo))
+	ipMap := make(map[string]struct{}, len(serversInfo))
+	for _, srv := range serversInfo {
+		if _, ok := ipMap[srv.IP]; ok {
+			continue
+		}
+		ipMap[srv.IP] = struct{}{}
+		addr := srv.IP + ":" + strconv.FormatUint(uint64(srv.StatusPort), 10)
+		addrWithTps = append(addrWithTps, addressWithType{addr: addr, tp: "tidb"})
```

Comment:
It's better to use the `TYPE` column of `TiDB_CLUSTER_INFO` table instead of defining individually (keep them consistent).

Reply:
ok~